Gemfile

```diff
@@ -1,7 +1,5 @@
-source 'http://rubygems.org'
+source 'https://rubygems.org'
 
-gem 'midwire_common'
```

Comment:
Why was `midwire_common` removed? I believe it contains some rake tasks for the release process.

Reply:
eh - I always like as few dependencies as possible. On the `master` branch `bundle exec rake --tasks` only lists those found in `lib/tasks/data.rake`

But I can include it again if you wish.